components/renku_data_services/project/api.spec.yaml

```diff
@@ -271,15 +257,13 @@ components:
         creation_date:
           $ref: "#/components/schemas/CreationDate"
         created_by:
-          $ref: "#/components/schemas/User"
+          $ref: "#/components/schemas/Member"
```

Comment:
I don't know when we changed this but the example for this field a few lines below this contains the members property. But the members property is not part of the spec. So we should update that to match.